content/react/pt/get-started.md

```diff
@@ -44,34 +44,34 @@ yarn start
 </div>
 
 Sendo estes os seguintes: testes automáticos (Jest), desenvolvimento de componentes (Storybook) e a aplicação em si.
 
 ![3 modalidades](/app-three-modalities.png)
 
 
-Dependendo de qual parte da aplicação que estamos a trabalhar, podemos querer executar um ou mais simultâneamente.
-Visto que neste caso, o foco é a criação de um componente de interface de utilizador simples, iremos cingir-nos somente á execução de Storybook.
+Dependendo de qual parte da aplicação que estamos trabalhando, podemos querer executar um ou mais simultâneamente.
+Visto que, neste caso, o foco é a criação de um componente de interface de usuário simples, continuaremos executando o Storybook.
 
 ## Reutilização CSS
 
 A Taskbox reutiliza elementos de design do tutorial de React e GraphQL
-[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), como tal não será necessária a criação de CSS neste tutorial. 
+[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), por isso não será necessária a criação de CSS neste tutorial. 
 
-Com isto o conteúdo do ficheiro LESS será compilado num único ficheiro CSS e incluido na aplicação.
+Desta forma o conteúdo do arquivo LESS será compilado em um único arquivo CSS e incluído na aplicação.
 
-O CSS compilado encontra-se disponível [aqui](https://github.com/hichroma/learnstorybook-code/blob/master/src/index.css) e pela convenção CRA(Create React App) será necessário copiar seu conteúdo para o seguinte ficheiro src/index.css.
+O CSS compilado encontra-se disponível [aqui](https://github.com/hichroma/learnstorybook-code/blob/master/src/index.css) e pela convenção CRA(Create React App) será necessário copiar seu conteúdo para o seguinte arquivo src/index.css.
 
-![Interface Utilizador Taskbox](/ss-browserchrome-taskbox-learnstorybook.png)
+![Interface Usuário Taskbox](/ss-browserchrome-taskbox-learnstorybook.png)
 
 <div class="aside">
-  Se for necessária alguma alteração aos elementos de estilo, os ficheiros LESS originais encontram-se disponíveis no repositório Github.
+  Se desejar alterar o estilo, os arquivos LESS originais encontram-se disponíveis no repositório Github.
 </div>
 
 ## Adicionar recursos
 
-Irá ser necessário adicionar também as pastas com o tipo de letra e ícones que se encontram disponíveis [aqui](https://github.com/hichroma/learnstorybook-code/tree/master/public) á pasta `public`. Ao adicionar estes elementos, a aplicação irá renderizar de forma algo estranha. 
-Mas isto é de esperar, visto que não iremos trabalhar na aplicação agora.
+Será necessário adicionar também as pastas com o tipo de letra e icones que se encontram disponíveis [aqui](https://github.com/hichroma/learnstorybook-code/tree/master/public) à pasta `public`. Ao adicionar estes elementos, a aplicação irá renderizar de forma um pouco estranha. 
+Mas isto é esperado, visto que não iremos trabalhar na aplicação agora.
```

Comment:
Phrasing sounds pretty much perfectly clear for all. Also "ícones" is well written. There's an accent in there

Reply:
Sorry, it's just a grammatical correction related to the verbs "ir" and "ser".